src/client/Client.h

```diff
@@ -1054,8 +1054,11 @@ class Client : public Dispatcher, public md_config_obs_t {
   int lookup_hash(inodeno_t ino, inodeno_t dirino, const char *name,
 		  const UserPerm& perms);
   int lookup_ino(inodeno_t ino, const UserPerm& perms, Inode **inode=NULL);
+  int lookup_ino_unlock(inodeno_t ino, const UserPerm& perms, Inode **inode=NULL);
```

Comment:
Please rename these methods that expect the `client_lock` to be locked to have a leading underscore and remove the `_unlock` suffix. i.e. `_lookup_ino`. Also, please make these methods private. (Many of the Client methods which expect the `client_lock` locked are public but we don't need to make the problem worse.)

Otherwise, the code looks good. Thanks!